test/benchmarks/micro/eli_interpreter.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * Example interpreter from Eli Bendersky's article "Computed goto for efficient dispatch tables".
+ * https://eli.thegreenplace.net/2012/07/12/computed-goto-for-efficient-dispatch-tables
+ *
+ * clang -target wasm32 eli_interpreter.c -O3 --no-standard-libraries -Xlinker --no-entry -Xlinker --export-all -o eli_interpreter.wasm
+ */
+
+#define OP_HALT     0x0
+#define OP_INC      0x1
+#define OP_DEC      0x2
+#define OP_MUL2     0x3
+#define OP_DIV2     0x4
+#define OP_ADD7     0x5
+#define OP_NEG      0x6
+
+0103050602020604
+OP_INC, OP_MUL2, OP_ADD7, OP_NEG, OP_DEC, OP_DEC, OP_NEG, OP_DIV2, OP_HALT
```

Comment:
These seem to be stray things. Perhaps move them into a comment?

Reply:
Should not be here...